lambda-buffers-compiler/src/LambdaBuffers/Compiler/KindCheck/Context.hs

```diff
@@ -15,8 +16,8 @@ import Prettyprinter (
  )
 
 data Context = Context
-  { _context :: M.Map Variable Kind
-  , _addContext :: M.Map Variable Kind
+  { _context :: M.Map (InfoLess Variable) Kind
+  , _addContext :: M.Map (InfoLess Variable) Kind
```

Comment:
I find this also a bit awkward:

1. Afacit `_context` is a mapping of 'qualified type names' to their `Kind`, and `_addContext` is afaict the mapping of 'type arguments introduced by a type abstraction' to their `Kind`,
2. `Variable` seems like an non-useful concept that hides what we really care about explicitly, mapping of 'fully qualified type names' to their `Kind` and mapping of `type abstraction arguments` to their `Kind`. Not to mention it confuses the reader that sees 'type variables', 'unification variables' and this 'variable'.

Imo the context is

```
data Context = Context {
  qtyNameKinds :: Map QTyName Kind,
  tyAbsArgsKinds :: Map (InfoLess VarName) Kind
}
```